.ci/opensearch/Dockerfile.opensearch

```diff
@@ -9,6 +9,6 @@ ARG SECURE_INTEGRATION
 HEALTHCHECK --start-period=20s --interval=5s --retries=2 --timeout=1s \
   CMD if [ "$SECURE_INTEGRATION" != "true" ]; \
   then curl --fail localhost:9200/_cat/health; \
-  else curl --fail -k https:/localhost:9200/_cat/health -u admin:admin; fi
+  else curl --fail -k https:/localhost:9200/_cat/health -u admin:myStrongPassword123!; fi
```

Comment:
Should this change be version based? `admin` for <2.12 and `myStrongPassword123!` for >=2.12.